### ISSUE TITLE
typings indicate default whereas the js code by default is export all

### ISSUE THREAD

**Issue by jf3096** [xx45/dayjs#139]
in your ts typings:

```typescript
declare function dayjs (config?: ConfigType): Dayjs

export default dayjs
```
in your distjs.min.js
```javascript
...
 module.exports = e() 
...
```

**Comment by jf3096** [comment]
this casuses error for my webstorm as well as my TS

**Comment by jf3096** [comment]
i know that in tsconfig, it can be solved using  "allowSyntheticDefaultImports": true. However, it will be a better practice if you generated code follow the standard.

**Comment by iamkun** [comment]
Thanks. Any suggestion here?

**Comment by iamkun** [comment]
@jf3096 Not sure how to fix this, shall I just remove the last line of code?
```diff
declare function dayjs (config?: ConfigType): Dayjs

- export default dayjs
```

Or change it to 
```diff
declare function dayjs (config?: ConfigType): Dayjs

- export default dayjs
+ export dayjs
```

**Comment by styfle** [comment]
Fixed in #150
